### PR TITLE
#174797511-Swagger Documentation

### PR DESCRIPTION
**Pull request by BeyFaith**
#### What does this PR do?
Configure swagger for API documentation
#### Description of Task to be completed?
- Creating swagger.json
- Installing swagger-UI-express
- Configure end point for documentation
#### How should this be manually tested?
- Clone this repo: `git clone https://github.com/atlp-rwanda/script-heroes-bn-backend.git`
- Switch working directory: `cd script-heroes-bn-backend`
- Checkout the branch: `git checkout  ch-setting-swagger-doc-#174797511`
- Install all dependencies : `npm install`
- Run your server:`node src/index.js`
- Then head to your browser and type in: `http://localhost:3000/api-docs`
> You should see documentation for endpoint.
#### What are the relevant pivotal tracker stories?
[#174797511](https://www.pivotaltracker.com/story/show/174797511)
#### Screenshots (if appropriate)
Screen Shot 2020-09-18 at 13 17 55

### CODE REVIEW

**Review by dushimeemma — APPROVED** [review]
add some other endpoints

File: src/index.js [hunk 1]
```diff
@@ -34,16 +36,18 @@ app.use(
...
-} else {
-  mongoose.connect('mongodb://localhost/conduit');
-  mongoose.set('debug', true);
-}
+// if (isProduction) {
+//   mongoose.connect(process.env.MONGODB_URI);
+// } else {
+//   mongoose.connect('mongodb://localhost/conduit');
+//   mongoose.set('debug', true);
+// }
```
Comment (kipropbrian) [inline src/index.js]: Do not leave comments. Either remove the code or fix the issue.

**Comment by kipropbrian** [comment]
Is there a link to online swagger documentation?

File: src/index.js (hunk 1 above)
Comment (BeyFaith) [reply src/index.js]: @kipropbrian I edited the code.

Comment (BeyFaith) [reply src/index.js]: https://app.swaggerhub.com/apis-docs/BeyFaith/Barefoot-Nomad/1.0.0
This is the link to the online documentation

**Review by Habinezajanvier — APPROVED** [review]
Good

**Comment by kipropbrian** [comment]
Add the link to swagger as part of the readme

**Pull request merged**